bin/push-gh-pages.sh

```diff
@@ -1,7 +1,13 @@
 #!/bin/bash
 
+# check out a detached head either from the environment var $GIT_COMMIT (which Jenkins
+# has), or using the last commit against HEAD (for local/non-Jenkins dev)
 git checkout ${GIT_COMMIT:=$(git rev-parse --short HEAD)} #detached head
+# add files from the newly created /public folder
 git add public
```

Comment:
I think this needs to be changed to release since that's what we're generating the site into now

Reply:
Good catch!